sdk/schemaregistry/azure-schemaregistry-avroserializer/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include *.md
+include azure/__init__.py
```

Comment:
For this particular package, you need to include all init that you exclude in the setup.py, in a way that the sdist contains all init at all levels (you can open the sdist to check)

Reply:
OOB discussion: this file is for sdist